build/KoreBuild.ps1

```diff
@@ -98,38 +98,18 @@ if (!($env:Path.Split(';') -icontains $dotnetLocalInstallFolder))
     $env:Path = "$newPath"
 }
 
-# wokaround for CLI issue: https://github.com/dotnet/cli/issues/2143
-$sharedPath = (Join-Path (Split-Path ((get-command dotnet.exe).Path) -Parent) "shared");
-(Get-ChildItem $sharedPath -Recurse *dotnet.exe) | %{ $_.FullName } | Remove-Item;
-
-# We still nuget because dotnet doesn't have support for pushing packages
-$nugetExePath = Join-Path $PSScriptRoot 'nuget.exe'
-if (!(Test-Path $nugetExePath))
-{
-    Invoke-WebRequest "https://dist.nuget.org/win-x86-commandline/v4.0.0-rc4/NuGet.exe" -OutFile "$PSScriptRoot/nuget.exe"
-}
-
 $makeFileProj = "$PSScriptRoot/KoreBuild.proj"
 $msbuildArtifactsDir = "$repoFolder/artifacts/msbuild"
 $msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.log"
 $msBuildResponseFile = "$msbuildArtifactsDir/msbuild.rsp"
 
-$preflightClpOption='/clp:DisableConsoleColor'
-$msbuildClpOption='/clp:DisableConsoleColor;Summary'
-if ("${env:CI}${env:APPVEYOR}${env:TEAMCITY_VERSION}${env:TRAVIS}" -eq "")
-{
-    # Not on any of the CI machines. Fine to use colors.
-    $preflightClpOption=''
-    $msbuildClpOption='/clp:Summary'
-}
-
```

Comment:
We disabled color and did the 'tee' on Linux to workaround  https://github.com/Microsoft/msbuild/issues/1792 which has been fixed since mid March.

MSBuild console logger doesn't emit much color in minimal mode, so I don't think log size is a major concern: on the test run for this PR, it added a total of 76 bytes: https://travis-ci.org/aspnet/KoreBuild/jobs/226232644.